enterprise/helpers.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""
+Helper functions for enterprise app.
+"""
+
+from .models import EnterpriseCustomerBrandingConfiguration
+
+
+def get_branding_info(provider_id=None):
+    """
+    Return the EnterpriseCustomer branding information.
+    """
+    info = None
+    try:
+        info = EnterpriseCustomerBrandingConfiguration.objects.filter(
```

Comment:
You can achieve this functionality by using the django's `first` method.
https://docs.djangoproject.com/en/1.10/ref/models/querysets/#first